src/routes/auth/twitter.router.ts

```diff
@@ -29,9 +29,11 @@ class TwitterRouter {
     }
 
     static async startMigration(ctx: Context): Promise<void> {
+        const app: string = getOriginApp(ctx, Settings.getSettings());
+
         return ctx.render('start', {
             error: false,
-            generalConfig: ctx.state.generalConfig,
```

Comment:
koa-views already injects the ctx.state into its rendering context, so this is both wrong and redundant